geopandas/tests/test_plotting.py

```diff
@@ -622,6 +644,28 @@ def setup_method(self):
         df_nan = GeoDataFrame({"geometry": t3, "values": [np.nan]})
         self.df3 = pd.concat([self.df, df_nan])
 
+    def test_autolim_false(self):
```

Comment:
A minor point - can you organise this by geometry type instead of True/False? To have `test_autolim_polygons` and `test_autolim_lines` ... which test both True/False options.

Reply:
Thanks for looking at it! Just to be sure do you mean something like:

```python
class TestLineStringPlotting:
    def test_autolim_lines(self):
    # test both for True and False
class TestPolygonPlotting:
    def test_autolim_polygons(self):
    # test both for True and False
```

Isn't it redundant with the class names?

Reply:
Ah, true. Forgot it is organised in these classes. Ignore the comment.

Reply:
OK.